pkgs/misc/stabber/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, fetchFromGitHub, autoreconfHook, pkgconfig, glib, expat
+, libmicrohttpd
+}:
+
+with stdenv.lib;
+
+stdenv.mkDerivation rec {
+  name = "stabber-${version}";
```

Comment:
Could you please use the commit date as package version (instead of the commit ID)?
This is mainly to be able to sort packages by their versions.
https://nixos.org/nixpkgs/manual/#sec-package-naming

Reply:
sure, thanks for the hint. i haven't seen that part in the manual yet.
package is updated.